internal/commands/repo/list.go

```diff
@@ -62,6 +62,10 @@ var (
 			s := fmt.Sprintf("%v", r.IsPrivate)
 			return s, len(s)
 		}},
+		{"AUTOMATED", func(r hub.Repository) (string, int) {
```

Comment:
nit: What about `"AUTOBUILD"` ?

Reply:
I just used what is in the API, zero opinion. If you want to change it to merge, go for it :)